specifications/api/swagger.yaml

```diff
@@ -3323,14 +3360,66 @@ components:
         - title: outputs without amounts
           <<: *transactionNoAmountOutputs
 
+    ApiAssetQuantity: &ApiAssetQuantity
+      type: object
+      required:
+        - quantity
+        - unit
+      properties:
+        quantity:
+          type: integer
+          example: 14
+        unit:
+          type: string
+          enum:
+            - assets
+
+    ApiMintData: &ApiMintData
+      type: object
+      required:
+        - receiving_address
+        - amount
+      properties:
+        receiving_address: *addressId
+        amount: *ApiAssetQuantity
+
+    ApiBurnData: &ApiBurnData
+      allOf:
+        - *ApiAssetQuantity
+
+    ApiMintBurnOperation: &ApiMintBurnOperation
+      type: object
+      oneOf:
+        - title: "mint"
+          properties:
+            mint: *ApiMintData
+        - title: "burn"
+          properties:
+            burn: *ApiBurnData
+
+    ApiMintBurnData: &ApiMintBurnData
+      type: object
+      required:
+        - asset_name
+        - operation
+        - policy_script_template
+      properties:
+        policy_script_template: *ScriptTemplateValue
```

Comment:
yes, for shelley mode it could do the trick, but I would not like to be inconsistent with what is for multisig. Both have `Script Cosigner` . Also, it is not small work to have `timelocking` slimmed script handled (slimmed script:= without RequireSignature constructor). There will be another job to merge those, ensure translation between stripts, align with what ApiDecodedTransaction serves, etc. So I am not in favor of introducing timelocking slim script PLUS cosigner#0 instead of normal script used (which is tested and works already). Small user benefit, huge work/maintanace burden